fdbclient/CoordinationInterface.h

```diff
@@ -236,4 +239,30 @@ struct ProtocolInfoRequest {
 	}
 };
 
+// Returns true if the cluster descriptor may be modified.
+struct CheckDescriptorMutableReply {
+	constexpr static FileIdentifier file_identifier = 7784299;
+	CheckDescriptorMutableReply() = default;
+	explicit CheckDescriptorMutableReply(bool value) : value(value) {}
+	bool value;
```

Comment:
Can we rename this to something a little more descriptive, like:

```suggestion
	bool isMutable;
```